core/controllers/story_viewer.py

```diff
@@ -33,12 +33,55 @@
 import utils
 
 
+SCHEMA_FOR_CLASSROOM_URL_FRAGMENT = {
```

Comment:
I think you can these schemas, you can introduce extra validator but try to use it from constants.ts.
https://github.com/oppia/oppia/blob/a0c63b07712a0cfb34a0cc5d4de8aaceeb709b9c/assets/constants.ts#L6350

Reply:
done